addons/survey/models/survey_user.py

```diff
@@ -243,7 +261,7 @@ def _save_line_matrix(self, question, old_answers, answers, comment):
             for row_key, row_answer in answers.items():
                 for answer in row_answer:
                     vals = self._get_line_answer_values(question, answer, 'suggestion')
-                    vals['matrix_row_id'] = row_key
+                    vals['matrix_row_id'] = int(row_key)
```

Comment:
this should be part of a separated commit

Reply:
Well not really.
It did not seem to break anything before my changes...

So I can't really make a "fix" commit that doesn't fix anything :D